torch_xla/experimental/pjrt.py

```diff
@@ -220,6 +233,49 @@ def _discover_tpu_master_worker_ip(device: torch.device):
   return _merge_replica_results(replica_results)
 
 
+@requires_pjrt
+def _run_singleprocess(fn: Callable[..., R],
+                       *args,
+                       start_method: str = 'spawn',
+                       master_port: int = 12355,
+                       **kwargs) -> Dict[int, R]:
+  """Runs `fn` on a single device core.
+
+  Spawns one process on a single physical device (e.g. TPU chip).
+
+  Args:
+    fn: Function to run on the device devices
+    args: args to pass to `fn`
+    start_method: The Python `multiprocessing` process creation method.
+      Default: `spawn`
+    kwargs: kwargs to pass to `fn`
+
+  Returns:
+    the result of calling `fn`.
+  """
+  with concurrent.futures.ProcessPoolExecutor(
+      max_workers=1,
+      mp_context=torch.multiprocessing.get_context(start_method)) as executor:
+    os.environ.setdefault('LOCAL_WORLD_SIZE', '1')
+
+    if device_type() == 'TPU':
+      tpu.configure_topology(0, 1)
```

Comment:
Should this be `tpu.configure_one_chip_topology`? That will set all of the necessary env vars.

Reply:
Done.